docs/building.md

```diff
@@ -0,0 +1,36 @@
+These are instructions to build emacs-ng on your workstation. First [check the requirements][compile-req].
+
+[compile-req]: https://emacs-ng.github.io/emacs-ng/getting-started/index.html#requirements
```

Comment:
Maybe we should pull the requirements and everything else that is related to building emacs-ng ?

Reply:
moved the build requirements in the new `building` section